tests/integration/istioio/trafficmanagement/ingress/scripts/troubleshooting.txt

```diff
@@ -35,8 +35,3 @@ $ kubectl logs -n istio-system $(kubectl get pod -l istio=ingressgateway \
 # $snippet inspect_secrets syntax="bash"
 $ kubectl -n istio-system get secrets
 # $endsnippet
-
```

Comment:
Why did you remove this? It seems it still exists at https://preliminary.istio.io/docs/tasks/traffic-management/ingress/secure-ingress-sds/#troubleshooting

Reply:
the container does not exist. I thought the docs are generated from this?

Reply:
I'm not sure... It seems to be hardcoded in the docs: https://github.com/istio/istio.io/blob/master/content/en/docs/tasks/traffic-management/ingress/secure-ingress-sds/index.md#troubleshooting

Well, if you are sure this container does exists, this change is good and the docs should be updated as well.

I was just confused on what this change relates to the title of this PR.

Reply:
ok, I'll update the doc. previously these doc tests were using the old helm install, so it wasn't really properly testing our docs since most users are on istioctl

Reply:
and this change was needed due to a difference between the two